beaker/session.py

```diff
@@ -231,6 +288,35 @@ def _set_serializer(self, data_serializer):
     def has_key(self, name):
         return name in self
 
+    def new_reads(self):
+        if self.migration_provider is None:
+            return False
+        
+        migration = self.migration_provider()
```

Comment:
ah I see, this allows us to read settings from consul all the time, without having to bounce the worker to get new settings